src/InputCheckbox/InputCheckbox.tsx

```diff
@@ -67,7 +68,12 @@ export const InputCheckbox = (p: InputCheckboxProps) => {
               <Spacer tiny={p.size === "small"} xsmall={p.size === "large"} />
 
               <Label htmlFor={key}>
-                <Text xsmall={p.size === "small"} small={p.size !== "small"} fill={[Color.Black, 700]}>
+                <Text
+                  xsmall={p.size === "small"}
+                  small={p.size !== "small"}
```

Comment:
I found this line confusing 😆 
How about

```suggestion
                  small={p.size === "large"}
```